cmd/minishift/cmd/start.go

```diff
@@ -255,13 +270,15 @@ func determineInsecureRegistry(key string) []string {
 	return append(s, defaultInsecureRegistry)
 }
 
-func startHost(libMachineClient *libmachine.Client) (*host.Host, string) {
+func startHost(libMachineClient *libmachine.Client, vmDriver string) *host.Host {
```

Comment:
> was previously suggested to remove the Viper call and pass it to the function...

Really. You are still in the cmd structure. In there I see no problem using Viper. What we have as a goal is to not let Viper/Cobra "leak" into anything under _pkg_. Within the _cmd_ I generally see no issues to access Viper.

> actually, it would be nicer if the whole config struct gets extracted,

Not sure what you mean.